specifyweb/frontend/js_src/lib/components/Statistics/index.tsx

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+
+import { useAsyncState } from '../../hooks/useAsyncState';
+import { commonText } from '../../localization/common';
+import { ajax } from '../../utils/ajax';
+import type { SpQueryField, Tables } from '../DataModel/types';
+import type { SerializedResource } from '../DataModel/helperTypes';
+import type { RA } from '../../utils/types';
+import { StatCategoryReturn, statsSpec } from './StatsSpec';
+import { BackendStatsResult, useStatAjaxHelper } from './utils';
+
+function useBackendApi(): BackendStatsResult | undefined {
+  const [backendStatObject] = useAsyncState(
+    React.useCallback(
+      async () =>
+        ajax<BackendStatsResult>('/statistics/collection/global/', {
+          method: 'GET',
+          headers: {
+            Accept: 'application/json',
+          },
+        }).then(({ data }) => data),
+      []
+    ),
+    false
+  );
+  return backendStatObject;
+}
+
+export function StatsPage(): JSX.Element {
+  const statsObjectCombined = statsSpec;
+  const backEndResult = useBackendApi();
+
+  return (
+    <div
+      style={{
+        display: 'flex',
+        height: '100%',
+        gap: '1rem',
+        flexDirection: 'column',
+        margin: '0 auto',
+        padding: '5%',
+        paddingTop: '0',
+      }}
+      className="bg-gray-200"
```

Comment:
Instead of hardcoding the background, look at what other pages do (they use a CSS variable)